update/files/multiFileWriter.go

```diff
@@ -0,0 +1,110 @@
+package files
+
+import (
+	"bufio"
+	"os"
+
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/ElrondNetwork/elrond-go/epochStart"
+	"github.com/ElrondNetwork/elrond-go/logger"
+	"github.com/ElrondNetwork/elrond-go/storage"
+	"github.com/ElrondNetwork/elrond-go/update"
+)
+
+var log = logger.GetOrCreate("update/files")
+
+type multiFileWriter struct {
+	exportFolder string
+
+	files       map[string]*os.File
```

Comment:
why not having the files slice be of type io.Closer? 
Also, for dataWriters we can have an interface like 
```GO
type DataWriter struct{
   WriteString(s string) (int, error)
   Flush() error
}
```
that will be automatically implemented by *bufio.Writer. Also, stubs can be easily created and this code can be tested in no-time.

Reply:
I do not know if stubbing even the writer is desirable. Some of the test would mean noting. Added io.Closer.

Reply:
done.